docs/quick-start-guide.md

```diff
@@ -62,9 +62,16 @@ import org.apache.spark.sql.CarbonSession._
 * Create a CarbonSession :
 
 ```
-val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession()
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>")
 ```
 
+Or another interface as:
+
+```
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>", "<local metastore path>")
+```
+NOTE:It is recommended to use the first method.
```

Comment:
We no need to  say  it is not recommended,  We can mention as if meta store path is not provided then it takes  default location as `../carbon.metastore`